.github/workflows/ci.linux.arm.yml

```diff
@@ -37,12 +37,12 @@ jobs:
           source /opt/rh/gcc-toolset-9/enable
           cmake -B build -D CMAKE_BUILD_TYPE=MinSizeRel -D PHOTON_BUILD_TESTING=ON \
             -D PHOTON_ENABLE_SASL=ON -D PHOTON_ENABLE_FUSE=ON -D PHOTON_ENABLE_EXTFS=ON
-          cmake --build build -j -- VERBOSE=1
+          cmake --build build -j $(nproc) -- VERBOSE=1
```

Comment:
What's the default value of nproc? Didn't see the definition.

Reply:
it's the number of processors in current system.